higher_health/forms.py

```diff
@@ -45,6 +53,67 @@ class HealthCheckQuestionnaire(forms.Form):
     latitude = forms.CharField(widget=forms.HiddenInput())
     longitude = forms.CharField(widget=forms.HiddenInput())
 
+    medical_sweating = forms.ChoiceField(
```

Comment:
I think we call these `symptoms` in other places, but `medical` is fine too

Reply:
I'm going to update to keep the same, thanks!